tests/msbr_processes.json

```diff
@@ -0,0 +1,103 @@
+{
+  "fuel": {
+    "extraction_processes": {
+      "core_inlet": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 9920000,
+        "volume": 0
+      },
+      "core_outlet": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 9920000,
+        "volume": 0
+      },
+      "heat_exchanger": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 9920000,
+        "volume": 13700000
+      },
+      "bypass": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 8928000,
+        "volume": 1000000000
+      },
+      "sparger": {
+        "capacity": 9920000,
+        "efficiency": {
+          "Kr": 0,
+          "Xe": 0,
+          "H": 0
```

Comment:
The sparger has an efficiency of zero?  Does that mean it's not removing anything?

Reply:
I believe so.